### PR TITLE
Searchbar

### PR DESCRIPTION
**Pull request by xingyif**
* added js file
* added a drop down calendar on main page search bar
* cleaned up index.html file
* overlaid search bar on top of images
        * changed width, height, top, margin, and color
        * TODO: change space between search bar columns, currently too wide

### CODE REVIEW

File: website/mainSearchPage/index.html [hunk 1]
```diff
@@ -242,12 +242,22 @@ <h2 class="featurette-heading">
...
+
+
+    <!------------------search bar calendar---------------->
+
+
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
```
Comment (Cheng-Lin) [inline website/mainSearchPage/index.html:249]: 你怎么把这些都给删了？

File: website/mainSearchPage/js/searchBarCalendar.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Created by yifanxing on 1/21/17.
+ */
+
+    $(document).ready(function() {
```
Comment (Cheng-Lin) [inline website/mainSearchPage/js/searchBarCalendar.js:5]: 这个应该不需要了吧？

File: website/mainSearchPage/index.html [hunk 3]
```diff
@@ -242,12 +242,22 @@ <h2 class="featurette-heading">
...
-            integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
-            crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script>
+
+
+    <script type="text/javascript" src="jquery-1.8.3.js"></script>
```
Comment (Cheng-Lin) [inline website/mainSearchPage/index.html:259]: type=“date”需要这些新的东西吗？不需要的话还是把这些都改回来的好

File: website/mainSearchPage/index.html (hunk 1 above)
Comment (xingyif) [reply website/mainSearchPage/index.html:249]: 我没删啊，换了个位置而已

File: website/mainSearchPage/js/searchBarCalendar.js (hunk 2 above)
Comment (xingyif) [reply website/mainSearchPage/js/searchBarCalendar.js:5]: 以后会需要

File: website/mainSearchPage/index.html (hunk 1 above)
Comment (Cheng-Lin) [reply website/mainSearchPage/index.html:249]: https://github.com/xingyif/YCTour/blob/searchbar/website/mainSearchPage/index.html 已不见了

Comment (xingyif) [reply website/mainSearchPage/index.html:249]: At the bottom of the file:
    

    
    <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script>

Comment (Cheng-Lin) [reply website/mainSearchPage/index.html:249]: bootstrap.min.js不见了啊

Comment (xingyif) [reply website/mainSearchPage/index.html:249]: doesn't make a diff.

Comment (Cheng-Lin) [reply website/mainSearchPage/index.html:249]: well, its part of bootstrap framework, and we are using it. could ended up having side affects if not there.

Comment (xingyif) [reply website/mainSearchPage/index.html:249]: already added back, thanks for catching it

**Pull request merged**